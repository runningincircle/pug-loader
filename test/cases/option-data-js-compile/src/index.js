// method compile via loader option
const tmpl = require('Views/widget.pug?{"a":10,"b":"abc"}');
// method render via query parameter
const html = require('Views/widget.pug?pug-render&{"a":20,"b":"def"}');

console.log(tmpl());
console.log(html);