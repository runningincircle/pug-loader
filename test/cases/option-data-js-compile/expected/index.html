<h1>Neu option data</h1><p>Hello world!</p><span>Param[a]: 10</span><span>Param[b]: abc</span>
<h1>Neu option data</h1><p>Hello world!</p><span>Param[a]: 20</span><span>Param[b]: def</span>